sentences/nl/weather_HassGetWeather.yaml

```diff
@@ -3,7 +3,11 @@ intents:
   HassGetWeather:
     data:
       - sentences:
-          - (wat|hoe) is (het weer|de weersvoorspelling) [buiten]
-          - (wat|hoe) is ((het weer|de weersvoorspelling);[( voor| in| volgens) <name>])
+          - wat voor weer is het[ nu][ buiten]
+          - (wat|hoe) is (het[ huidige] weer|de[ huidige] weersvoorspelling)[ buiten]
```

Comment:
Doesn't this require context?

(Sorry, I'm not extremely profound with intents, trying to learn).